src/api/test-helpers/applications.ts

```diff
@@ -71,10 +71,38 @@ const update = async ({ context, applicationId, data }: TestHelperApplicationUpd
   }
 };
 
+/**
+ * deleteAll test helper
+ * Get all accounts and delete them.
+ * @param {Object} KeystoneJS context API
+ * @returns {Array} Accounts that have been deleted
```

Comment:
Not a blocker, can fix in the next PR - this says accounts instead of applications